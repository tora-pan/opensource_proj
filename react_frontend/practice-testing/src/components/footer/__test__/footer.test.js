import React from 'react';
import Footer from '../Footer';

import { shallow } from 'enzyme';

describe('<Header />', () => {
	it('Footer Component', () => {
		const wrapper = shallow(<Footer />);
		expect(
			wrapper.containsMatchingElement(<h1>This is the footer.</h1>)
		).toBe(true);
	});
	it('footer component matches snapshot', () => {
		const wrapper = shallow(<Footer />);
		expect(wrapper).toMatchSnapshot();
	});
});
