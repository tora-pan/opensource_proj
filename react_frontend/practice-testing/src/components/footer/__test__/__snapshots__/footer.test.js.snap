// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> footer component matches snapshot 1`] = `ShallowWrapper {}`;
